README.md

```diff
@@ -91,7 +91,10 @@ level = "info"
 [data]
 # Folder for buffering received data
 path = "/data/carbon-clickhouse/"
-# Rotate (and upload) file interval.
+# Rotate (and upload) file iniciated on size and interval
+# Rotate (and upload) file size
+chunk-switch-size = 0
```

Comment:
Bulk suggestion:
```suggestion
chunk-max-size = 0
```

Reply:
It's not max size, size can be some bigger (check granularity is 100 ms) :-) If this size reached, but interval not reached - rotate initiated.

Reply:
It would be more semantically close to, e.g., logrotate `maxsize` parameter https://manpages.debian.org/jessie/logrotate/logrotate.8.en.html

> maxsize size
>    Log files are rotated when they grow bigger than size bytes even before the additionally specified time interval. 

And, technically, will mean the same behavior.  
It's on you to decide, though. :+1: